examples/proc_combined_ODNP.py

```diff
@@ -264,5 +264,5 @@
     fl.plot(np.real(s_int['power',:-3]),'ko',capsize=6,alpha=0.3)
```

Comment:
shouldn't this be an example in the procScripts repo?

Reply:
yes replace with raw_odnp.py

Reply:
I replaced proc_ODNP_rough with the raw_odnp not sure why we have both proc_combined_odnp and proc_odnp_rough we should only have rough processing to view raw data and finer processing done in proc_scripts